bitmind/base/utils/weight_utils.py

```diff
@@ -157,6 +157,9 @@ def process_weights_for_netuid(
 
     # Find all non zero weights.
     non_zero_weight_idx = np.argwhere(weights > 0).squeeze()
+    if non_zero_weight_idx.ndim == 0:
+        non_zero_weight_idx = non_zero_weight_idx.reshape((1,))
+
```

Comment:
this is a fix for an issue i raised a while back for the bittensor-subnet-tempalte repo:
https://github.com/opentensor/bittensor-subnet-template/issues/93